docs/howto/labels.md

```diff
@@ -0,0 +1,116 @@
+# How to define labels
+
+**Note**
+The term label may designate several concepts. Please check the
+[`ml:label` definition](/definitions/label).
+
+This documentation page covers the following topics:
+
+- Defining a [`ml:label`](/definitions/label), including a [class](/definitions/class).
+- Defining a label following the general definition "a kind of tag to
+  idendify an object" or identifying tag.
+
+## Defining a `ml:label`
+
+To define a field as being a [`ml:label`](/definitions/label), one must just
+add `ml:label` to its `dataType`s, for example:
+
+```json
+{
+    "@type": "ml:RecordSet",
+    "name": "weather_report",
+    ...
+    "field": [
+        ...,
+        {
+            "@type": "ml:Field",
+            "name": "air_temperature_c",
+            "description": "The air temperature, taken in the shadow.",
+            "dataType": ["sc:Number", "ml:label"],
+            "source": "..."
+        },
+    ],
+    ...
+}
+```
+
+In the above example, the `air_temperature_c` field is declared as being a
+`ml:label`, on top of being a number. This means that the dataset has
+originally been collected to train models that could predict the air
+temperature in ºC, given other parameters. Note that this does not prevent
+someone from using that same dataset for other purposes as predicting air
```

Comment:
Does this mean that a field labeled as a label can also be used e.g. as a feature?